Formula/l/ludusavi.rb

```diff
@@ -0,0 +1,17 @@
+class Ludusavi < Formula
+  desc "Backup tool for PC game saves"
+  homepage "https://github.com/mtkennerly/ludusavi"
+  url "https://github.com/mtkennerly/ludusavi/archive/refs/tags/v0.28.0.tar.gz"
+  sha256 "c5aa44b95326028df48c14efd9d561991c0eb19c66bb72b8f21394a04439d1fe"
+  license "MIT"
+
+  depends_on "rust" => :build
```

Comment:
`pokgconf` seems needed during the build process.
```suggestion
  depends_on "pkgconf" => :build
  depends_on "rust" => :build
```

Reply:
I'll include it, thanks.